src/components/line-chart-audification/line-chart-audification.component.ts

```diff
@@ -49,8 +49,13 @@ export class LineChartAudificationComponent implements AudificationPreference, O
     return this.host.data;
   }
 
-  get measureNames() {
-    return this.host.measureNames;
+  get legendItems() {
+    return this.host.legendItems;
+  }
+
+  get currentMeasureName() {
+    // TODO: replace 0
```

Comment:
With what?

Reply:
I temporarily made it always get the first legend item, but later it should get the selected measure.
All TODO comments will be resolved by the time I mark this PR non-draft (and ready for review).